.travis.yml

```diff
@@ -28,10 +28,10 @@ cache: yarn
 
 before_install:
   - export $(dbus-launch)
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - travis_retry curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
Does anybody measure the difference between `yarn install` and `npm install`? I guess it is a very little as compared with functional test execution time. So, we can use a standard `npm install`.